lunes_cms/cms/models/document.py

```diff
@@ -37,6 +37,20 @@ class Document(models.Model):
         default="",
         verbose_name=_("article"),
     )
+    plural = models.CharField(
+        max_length=255,
+        verbose_name=_("plural"),
+        blank=True,
+        default="",
+        help_text=_("Only required if 'Substantiv' is selected as word type."),
+    )
+    plural_article = models.IntegerField(
+        choices=Static.article_choices,
+        default=4,
```

Comment:
```suggestion
        null=True,
```
I think a default value `None` makes much more sense for non substantives.

Reply:
Yes, fair point